Vungle/VungleRouter.m

```diff
@@ -32,6 +32,8 @@
 static NSString *const kVungleBannerDelegateKey = @"bannerDelegate";
 static NSString *const kVungleBannerDelegateStateKey = @"bannerState";
 
+const CGSize kVGNMRECSize = {.width = 300.0f, .height = 250.0f};
```

Comment:
Because MOPUB_MEDIUM_RECT_SIZE is deprecated in MoPub 5.8.0, and if we use MOPUB_MEDIUM_RECT_SIZE, it will appear a warning,  so I define our own MREC Size constant.